airflow/www_rbac/templates/airflow/dags.html

```diff
@@ -224,14 +224,15 @@ <h2>DAGs</h2>
 
 {% block tail %}
   {{ super() }}
-  <script src="{{ url_for('static', filename='d3.v3.min.js') }}"></script>
-  <script src="{{ url_for('static', filename='jquery.dataTables.min.js') }}"></script>
-  <script src="{{ url_for('static', filename='bootstrap-toggle.min.js') }}"></script>
-  <script src="{{ url_for('static', filename='bootstrap3-typeahead.min.js') }}"></script>
+  <script src="{{ url_for_asset('d3.min.js') }}"></script>
+  <script src="{{ url_for_asset('jquery.dataTables.min.js') }}"></script>
+  <script src="{{ url_for_asset('dataTables.bootstrap.min.js') }}"></script>
```

Comment:
New JS file -why's that?

Reply:
The existing [`jquery.dataTables.min.js`](https://github.com/apache/incubator-airflow/blob/master/airflow/www_rbac/static/jquery.dataTables.min.js) is a combination of jQuery Data Tables and Data Tables Bootstrap (for stylings), which is exactly same as this. They don't ship that in a single build as npm package, so needed to add both.